docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  fhir:
+    depends_on:
+      - mongo
+    build:
+      context: fhir
+      dockerfile: Dockerfile
+    environment:
+      MONGO_HOSTNAME: mongo
+      MONGO_DB_NAME: mongo_on_fhir
+      SERVER_PORT: 3000
+    ports:
+      - "3000:3000"
+    volumes:
+      - ./fhir/scripts:/srv/src/scripts
+      - ./.eslintrc:/srv/src/.eslintrc
```

Comment:
Why does the `.eslintrc` file need to be specifically mapped over?  Was that just for testing?  That file shouldn't change often